refactor: new CLI commands documentation (#413)
NodeSecure/cli: docs/cli/lang.md

## 📝 Command `lang`

> The `lang` command is used to set the language for the NodeSecure CLI tool. It allows users to choose their preferred language for the CLI interface.

Reviewer: ```suggestion
The `lang` command allows you to set your preferred language. Use this command to customize the language used in the CLI or Web interfaces.
```
